plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -97,7 +98,7 @@ type indexStat struct {
 type Elasticsearch struct {
 	Local                      bool            `toml:"local"`
 	Servers                    []string        `toml:"servers"`
-	HTTPTimeout                config.Duration `toml:"http_timeout"`
+	HTTPTimeout                config.Duration `toml:"http_timeout" deprecated:"1.29.0;use 'timeout' instead"`
```

Comment:
Please also mark this config option as deprecated in the sample config conform https://github.com/influxdata/telegraf/blob/master/docs/developers/DEPRECATION.md#deprecate-options